src/register/messages.jsx

```diff
@@ -126,6 +126,11 @@ const messages = defineMessages({
     defaultMessage: 'Enter a valid name',
     description: 'Validation message that appears when fullname contain URL',
   },
+  'name.validation.length.message': {
+    id: 'name.validation.message',
+    defaultMessage: 'Full name can not be longer than 255 symbols',
```

Comment:
Do we need product approval on this message copy? If we don't, then maybe we can have an error message as for username field.
`Full name must be between 1 and 255 characters`?

Not sure but please let me know if my comment makes sense